libfreerdp/locale/timezone.c

```diff
@@ -1498,7 +1498,9 @@ static UINT64 freerdp_windows_gmtime()
 	UINT64 windows_time;
 
 	time(&unix_time);
-	windows_time = ((UINT64) unix_time * 10000000) + 621355968000000000ULL;
```

Comment:
I kind of agree it wasn't bad in its original state, don't see much of a point in splitting it over multiple statements like that.